docs/requirements.md

```diff
@@ -1,51 +1,140 @@
+#### Transactions
+{:.no_toc}
+- Use a single primary region for all operations with `Strong` or `Bounded Staleness` consistency.
 
-## CosmosDB database requirements
+#### Analytics
+{:.no_toc}
+- N/A (Thee are no Cosmos DB-specific requirements since Cosmos DB always returns committed records.)
 
-In your Azure CosmosDB account, you must set the **default consistency level** to **Strong**.
+</div>
 
-Consensus Commit, the ScalarDB transaction protocol, requires linearizable reads. By setting the **default consistency level** to **Strong**, CosmosDB can guarantee linearizability.
+<div id="Cassandra" class="tabcontent" markdown="1">
 
-For instructions on how to configure this setting, see the official documentation at [Configure the default consistency level](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-manage-consistency#configure-the-default-consistency-level).
+#### Transactions
+{:.no_toc}
+- Use a single primary cluster for all operations. (No reads and writes in non-primary clusters.)
+- Use `batch` or `group` for `commitlog_sync`.
+- (In case you use Cassandra-compatible databases) Use Cassandra-compatible databases that properly support Lightweight Transactions (LWT).
 
-## JDBC database recommendations
+#### Analytics
+{:.no_toc}
+- N/A (Thee are no Cassandra-specific requirements since Cassandra always returns committed records.)
 
-In ScalarDB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code by using the `Operation.withConsistency()` method. In addition, the consistency level depends on the setup of your JDBC database.
+</div>
+</div>
 
-For example, if you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas. On the other hand, if you perform all operations against a single master instance, the consistency will be linearizable.
+## Recommendations
 
-With this in mind, you must perform all operations or transactions against a single master instance so that you can achieve linearizability and avoid worrying about consistency issues in your application. In other words, ScalarDB does not support read replicas. 
+ScalarDB recommends each underlying database to be configured properly for high performance and high availability. The following items are some knobs and configurations to update.
 
 {% capture notice--info %}
 **Note**
 
-You can still use a read replica as a backup and standby even when following this guideline.
+ScalarDB can be seen as an application of underlying databases, so it is worth trying other knobs and configurations that are commonly effective.
```

Comment:
Revised to use active voice. I also revised the `commonly effective` ending since I felt it was a little awkward, but please let me know if my suggestion is incorrect.

```suggestion
ScalarDB can be seen as an application of underlying databases, so you may want to try updating other knobs and configurations that are commonly used to improve efficiency.
```